pkg/services/alerting/notifiers/pushover.go

```diff
@@ -42,10 +42,85 @@ func init() {
           'updown',
           'none'`
 
+	soundOptions := []alerting.SelectOption{
+		{
+			Value: "default",
+			Label: "Default",
+		},
+		{
+			Value: "pushover",
+			Label: "Pushover",
+		}, {
+			Value: "bike",
+			Label: "Bike",
+		}, {
+			Value: "bugle",
+			Label: "Bugle",
+		}, {
+			Value: "cashregister",
+			Label: "Cashregister",
+		}, {
+			Value: "classical",
+			Label: "Classical",
+		}, {
+			Value: "cosmic",
+			Label: "Cosmic",
+		}, {
+			Value: "falling",
+			Label: "Falling",
+		}, {
+			Value: "gamelan",
+			Label: "Gamelan",
```

Comment:
If you happen to be unfamiliar, Gamelan music is pretty amazing: https://www.youtube.com/watch?v=UEWCCSuHsuQ